lib/events/dynamoevents/dynamoevents.go

```diff
@@ -189,6 +189,9 @@ type Log struct {
 	// readyForQuery is used to determine if all indexes are in place
 	// for event queries.
 	readyForQuery *atomic.Bool
+
+	// isProvisioned tracks if the table has provisioned capacity or not.
```

Comment:
Can we rename this? It's not totally clear that this is related to the billing mode. Maybe `billingModeIsProvisioned` or something?

Reply:
done